### ISSUE TITLE
Gesture Commands do not fire in TitleView

### ISSUE THREAD

**Issue by davidortinau** [xamarin/Xamarin.Forms#3763]
### Description
I can use a `TapGestureRecognizer` on my `Image` within `TitleView`, but only with the `Tapped` event. When using `Command` it doesn't fire.

**Works:**
```
<NavigationPage.TitleView>
        <StackLayout Orientation="Horizontal" VerticalOptions="Center" Spacing="10">
            <Image Source="iconXamagon.png">
                <Image.GestureRecognizers>
                    <TapGestureRecognizer Tapped="Handle_Tapped"/>
                </Image.GestureRecognizers>
            </Image>
            <Label Text="3.2.0" FontSize="16" TextColor="Black" VerticalTextAlignment="Center" />
        </StackLayout>
        
    </NavigationPage.TitleView>
```

**Does NOT Work:**
```
<NavigationPage.TitleView>
        <StackLayout Orientation="Horizontal" VerticalOptions="Center" Spacing="10">
            <Image Source="iconXamagon.png">
                <Image.GestureRecognizers>
                    <TapGestureRecognizer  
                            Command="{Binding TapCommand, Mode=OneWay}"
                            CommandParameter="https://www.xamarin.com"/>
                </Image.GestureRecognizers>
            </Image>
            <Label Text="3.2.0" FontSize="16" TextColor="Black" VerticalTextAlignment="Center" />
        </StackLayout>
        
    </NavigationPage.TitleView>
```

The `TapCommand` above works elsewhere on the page. I would like it to work in this scenario also.

### Basic Information

- Version with issue: 3.2.0-pre3
- Tested iOS and Android

**Comment by AmrAlSayed0** [comment]
Related #3596, here you mention that normal events work which was not mentioned in the original issue.

**Comment by samhouts** [comment]
closed by #3823
